### ISSUE TITLE
No Add New Connection window

### ISSUE THREAD

**Issue by tylercollier** [redis/RedisDesktopManager#2186]
I'm on Ubuntu 12.04 LTS. Installation seems fine. When I launch Redis Desktop Manager, it boots up immediately just fine. But when I click Add New Connection, it doesn't show any 'new connection' window, like what's shown [on the quick start guide](https://github.com/uglide/RedisDesktopManager/wiki/Quick-Start).

If I run it from the command line, and repeat the steps above, here's what I see:

```
$ /usr/share/redis-desktop-manager/bin/rdm 
QObject::connect: No such slot ConnectionWindow::clicked()
QObject::connect:  (sender name:   'okButton')
QObject::connect:  (receiver name: 'connectionDialog')
```

**Comment by uglide** [comment]
Thanks for feedback. I will investigate this problem.

**Comment by jnankin** [comment]
Any movement here?  I have this as well.

**Comment by jnankin** [comment]
This actually does work for me.  The issue was my desktop was larger than my screen size and the window was getting lost in another part of the desktop that i couldnt see.

**Comment by uglide** [comment]
Possible duplicate of #1293

**Comment by uglide** [comment]
Cannot reproduce on latest 0.7.9.X build. Seems that issue was fixed like #1283
